crates/scroll/consensus/src/validation.rs

```diff
@@ -189,3 +314,501 @@ fn validate_against_parent_timestamp<H: BlockHeader>(
     }
     Ok(())
 }
+
+/// Validates the gas limit of the block against the parent.
+#[inline]
+fn validate_against_parent_gas_limit<H: BlockHeader>(
+    header: &H,
+    parent: &H,
+) -> Result<(), ConsensusError> {
+    let diff = header.gas_limit().abs_diff(parent.gas_limit());
+    let limit = parent.gas_limit().saturating_div(GAS_LIMIT_BOUND_DIVISOR);
```

Comment:
May `checked_div` be better here?

Reply:
I replaced it with plain div `/`, I saturating div here wasn't useful and checked_div returns `None` if the divisor is 0, which is guaranteed not to be the case here.

Reply:
I think I can even just use plain div, checked will return `None` if the divisor is 0, which in this case it wouldn't be.

Reply:
Good idea.